torchinductor/sizevars.py

```diff
@@ -348,6 +348,7 @@ def codegen_shape_tuple(self, shape):
         return f"({', '.join(parts)})"
 
 
+@functools.lru_cache(None)
```

Comment:
Any other hot functions in this file?

Reply:
Not as hot I believe.

We can shave off about another 2-3% by caching on `simplify_sizes_with_ranges`, but it requires a bit more code change due to taking in dicts.

Reply:
We should have some limit on the cache size other than `None`, perhaps 
```suggestion
@functools.lru_cache(128)
```

Reply:
Seems fine to me.

With `cache_size=None`, on densenet:

```
CacheInfo(hits=32555, misses=2486, maxsize=None, currsize=2485)
``

With `cache_size=256`

```
CacheInfo(hits=32023, misses=3072, maxsize=256, currsize=256)
```